Frends.AzureBlobStorage.DeleteBlob/Frends.AzureBlobStorage.DeleteBlob/Definitions/Input.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureBlobStorage.DeleteBlob
+{
+    /// <summary>
+    /// Input-class for DeleteBlob-task.
+    /// </summary>
+    public class Input
+    {
+        /// <summary>
+        /// Name of the blob to delete
+        /// </summary>
+        [DisplayName("Blob name")]
+        [DefaultValue("TestFile.xml")]
+        [DisplayFormat(DataFormatString = "Text")]
+        public string BlobName { get; set; }
+
+        /// <summary>
+        /// Delete blob only if the ETag matches.
+        /// Leave empty if verification is not needed.
+        /// </summary>
+        [DisplayName("Verify ETag When Deleting")]
+        [DefaultValue("0x9FE13BAA323E5A4")]
+        [DisplayFormat(DataFormatString = "Text")]
+        public string VerifyETagWhenDeleting { get; set; }

```

Comment:
This looks like an option, should it be in the options class? Not sure, but just a thought.

Reply:
Moved to options.